03week/towersOfHanoi.js

```diff
@@ -7,38 +7,78 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+// object containing stack info
 let stacks = {
-  a: [4, 3, 2, 1],
+  a: [4,3,2,1],
   b: [],
   c: []
 };
 
+// prints the current board
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
-
-function movePiece() {
-  // Your code here
-
+// moves the disc from startStack to endStack
+function movePiece(startStack, endStack) {
+  let stack1 = stacks[startStack];
+  let stack2 = stacks[endStack];
+  let takeDisc = stack1.pop();
+  let dropDisc = stack2.push(takeDisc);
 }
 
-function isLegal() {
-  // Your code here
-
+function isLegal(startStack, endStack) {
+  let stack1 = stacks[startStack];
```

Comment:
this allows for a stack to be both starting and ending stack, which is not wrong just odd